docs/docs/markers.mdx

```diff
@@ -0,0 +1,291 @@
+---
+id: markers
+sidebar_label: Markers
+title: Markers
+description: Read more about how to mark points of interest in dialogues using logical expressions.
+abstract: Markers are logical expressions that allow you to describe points in the dialogue that you are interested in. 
+---
+
+:::caution
+
+This feature is currently experimental and might change or be removed in the future. Share your feedback in the forum to help us make it production-ready.
+
+:::
+
+Markers allow you to describe points in a dialogue that you are interested in identifying. In Rasa, a dialogue is represented as a sequence of events, which include bot actions that were executed (`ActionExecuted`), intents that were detected (`UserUttered`), and slots that were set (`SlotSet`). A point of interest in a dialogue can thus be expressed as a logical expression that can be evaluated against the sequence of dialogue events. When this logical expression is met, we "mark" that point in the dialogue for further analysis or inspection.
```

Comment:
The start of the paragraph talks about what "you" can do. While the bottom talks about what "we" can do. Since the docs are about the user who reads it, I propose using "you" over "we" for consistency. 

`we "mark" that point in the dialogue` -> `you can "mark" that point in the dialogue`.